sample-hacep-project/sample-hacep-project-kjar/pom.xml

```diff
@@ -17,13 +18,15 @@
         <dependency>
             <groupId>org.kie</groupId>
             <artifactId>kie-remote</artifactId>
-            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.drools</groupId>
             <artifactId>drools-model-compiler</artifactId>
-            <version>${version.org.kie}</version>
         </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-workbench-models-datamodel-api</artifactId>
```

Comment:
I don't understand why you added this dependency. It works for me even without it.